backend/apps/users/models.py

```
@@ -0,0 +1,117 @@
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+from django.db import models
+from cloudinary.models import CloudinaryField
+
+
+class UserManager(BaseUserManager):
+    """ Custom user manager """ 
+    def create_user(self, email, name, password=None, user_type=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_user`

(ANN201)